### PR TITLE
RestrictedFunctions: Add stats_get_csv()

### LINKED ISSUE #461: Add stats_get_csv to restricted functions
## What problem would the enhancement address for VIP?

Every time `stats_get_csv()` is called, it stores the data in the option `stats_cache`, which is where it can be polluted, as Jetpack relies on this value: https://github.com/Automattic/jetpack/blob/29b6f22dd57d0a5e8967fef73279dfdc575b820d/modules/stats.php#L1490. 

## Describe the solution you'd like

This function should not be relied on and a custom solution (e.g. stats endpoint) should be used instead.

## What code should be reported as a violation?

```
$popular = stats_get_csv( 'postviews', [ 'days'  => 2,   'limit' => 20 ] );
```

## What code should *not* be reported as a violation?

```
$popular = custom_stats_get_csv( 'postviews', [ 'days'  => 2,   'limit' => 20 ] );
```

### PR DESCRIPTION
**Pull request by rebeccahum**
Fixes #461.

### CODE REVIEW

File: WordPressVIPMinimum/Sniffs/Functions/RestrictedFunctionsSniff.php [hunk 1]
```diff
@@ -268,6 +268,13 @@ public function getGroups() {
 					'delete_site_option',
 				],
 			],
+			'stats_get_csv' => [
+				'type'      => 'error',
+				'message'   => 'Using `%s` outside of Jetpack context pollutes the stats_cache option. We recommend building a custom function instead of relying on it.',
```
Comment (GaryJones) [inline WordPressVIPMinimum/Sniffs/Functions/RestrictedFunctionsSniff.php]:
```suggestion
				'message'   => 'Using `%s` outside of Jetpack context pollutes the stats_cache entry in the wp_options table. We recommend building a custom function instead.',
```

**Comment by GaryJones** [comment]
Needs unit tests.

**Comment by rebeccahum** [comment]
@GaryJones Whoops. Done!

File: WordPressVIPMinimum/Sniffs/Files/IncludingFileSniff.php [hunk 2]
```diff
@@ -31,7 +31,6 @@ class IncludingFileSniff extends AbstractFunctionRestrictionsSniff {
 		'get_stylesheet_directory',
 		'get_template_directory',
 		'locate_template',
-		'get_parent_theme_file_path'
```
Comment (GaryJones) [inline WordPressVIPMinimum/Sniffs/Files/IncludingFileSniff.php]: Think this came across from another PR?

**Pull request merged**